publish/publish.proj

```diff
@@ -23,7 +23,7 @@
 
   <Target Name="PublishToAzure"
           Condition="'$(Finalize)' != 'true'"
-          DependsOnTargets="GatherPublishToAzureBinaries;GatherPublishToAzureInstallers">
+          DependsOnTargets="GatherPublishToAzureBinaries;GatherPublishToAzureInstallers;GatherPublishToAzureChecksums">
```

Comment:
The only benefit I can see from this was reuse of the same parameters but we don't need that anymore.